Build broken on Falcon

### Description of the enhancement or error report
I get the following compile error on Falcon:
```
moose/modules/phase_field/include/ics/SmoothCircleFromFileIC.h:43:48: error: array must be initialized with a brace-enclosed initializer
   std::array<int, 4> _col_map = {-1, -1, -1, -1};
```

### Rationale for the enhancement or information for reproducing the error
Compile with the `moose-dev-gcc` module.
I followed the advice of this thread, and added a second set of braces, and that works:
https://stackoverflow.com/questions/32912921/whats-wrong-with-this-inline-initialization-of-stdarray

### Identified impact
Build is broken.

--- a/modules/phase_field/include/ics/SmoothCircleFromFileIC.h
+++ b/modules/phase_field/include/ics/SmoothCircleFromFileIC.h
@@ -40,7 +40,8 @@ protected:
     Z,
     R
   };
-  std::array<int, 4> _col_map = {-1, -1, -1, -1};
+  // Double braces in array initializer are workaround for bug in gcc 4.9.2
+  std::array<int, 4> _col_map = {{-1, -1, -1, -1}};
   std::vector<std::vector<Real>> _data;
   FileName _file_name;
   MooseUtils::DelimitedFileReader _txt_reader;

